tests/unit/weight-test-modules-test.js

```diff
@@ -0,0 +1,15 @@
+import { module, test } from 'qunit';
+import weightTestModules from 'ember-exam/test-support/-private/weight-test-modules';
+
+
+module('Unit | weight-test-modules', () => {
+  test('should sort a list of file paths by weight', function(assert) {
+    const listOfModules = ['/jshint/test-1-test', '/acceptance/test-1-test', '/unit/test-1-test', '/integration/test-1-test', 'test-1-test']
```

Comment:
Can we format this so each item is on a newline? This makes it _much_ easier to scan top to bottom rather than left to right. Same goes for all below.